rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_reader :book, :person
+  attr_accessor :date
```

Comment:
- [ ] Please note that the `: book, : person`  should be attr_accessor and not attr_reader because we need both  attr_reader and attr_writer , kindly fix that